packages/jsxstyle-utils/src/getStyleCache.ts

```diff
@@ -14,17 +28,17 @@ function alreadyInjected() {
   );
 }
 
-function getStringHash(key) {
+function getStringHash(key: string, props: {}) {
   return '_' + stringHash(key).toString(36);
 }
 
 export default function getStyleCache() {
   let _classNameCache = {};
   let getClassNameForKey = getStringHash;
-  let onInsertRule = null;
+  let onInsertRule: InsertRuleCallback;
   let pretty = false;
 
-  const styleCache = {};
+  const styleCache: StyleCache = <any>{};
```

Comment:
`({} as any)` is easier to read.  I don't often see `<>` used for casting.

But more holistically, I don't think you need to cast this at all - it's just sidestepping the type checker.  Can you just refactor it to

```typescript
const styleCache: StyleCache = {
  reset() {
    _classNameCache = {}
  }
  // ...
}
```

Reply:
Yeah, refactoring is the way to go I think. I was trying to refactor as little as possible to get a first version out ASAP, but in this case it’s quick and easy.